### PR TITLE
Telegram notification manager

### PR DESCRIPTION
**Pull request by Raimon93**
Telegram module for the notification manager

### CODE REVIEW

File: functions/Mynt.Core/NotificationManagers/TelegramNotificationManager.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    {
+        
+        private readonly string TelegramWebhookUrl = "https://api.telegram.org/botTOKEN/sendMessage";
+        
+        public async Task<bool> SendNotification(string message)
+        {
+            try
+            {
+                
+            var payload = new Payload() { text = message };
```
Comment (sthewissen) [inline functions/Mynt.Core/NotificationManagers/TelegramNotificationManager.cs]: This isn't being used.

File: functions/Mynt.Core/NotificationManagers/TelegramNotificationManager.cs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            var payloadJson = JsonConvert.SerializeObject(payload);
```
Comment (sthewissen) [inline functions/Mynt.Core/NotificationManagers/TelegramNotificationManager.cs]: This isn't being used.

File: functions/Mynt.Core/NotificationManagers/TelegramNotificationManager.cs [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+            }
+        }
+
+        public async Task<bool> SendTemplatedNotification(string template, params object[] parameters)
+        {
+            var finalMessage = string.Format(template, parameters);
+           return await SendNotification(finalMessage);
+        }
+
+        public class Payload
```
Comment (sthewissen) [inline functions/Mynt.Core/NotificationManagers/TelegramNotificationManager.cs]: This entire class is not used for the Telegram side of things.

**Comment by Raimon93** [comment]
Changes made

**Pull request merged**